plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/preferences/PrefPageProjectNetworkProfiles.java

```diff
@@ -146,10 +146,14 @@ protected DBWNetworkProfile createNewProfile(@Nullable DBWNetworkProfile sourceP
                 profileName
             );
 
-            if (CommonUtils.isEmptyTrimmed(profileName)) {
+            if (CommonUtils.isEmpty(profileName)) {
```

Comment:
Bring `isEmptyTrimmed` back - otherwise, profiles with spaces only will pass through and get created